src/vs/workbench/contrib/terminal/browser/xterm/decorationAddon.ts

```diff
@@ -79,78 +96,69 @@ export class DecorationAddon extends Disposable implements ITerminalAddon {
 		if (!capability) {
 			return;
 		}
-		this._commandListener = capability.onCommandFinished(c => {
-			//TODO: remove when this has been fixed in xterm.js
-			if (!isWindows && c.command === 'clear') {
```

Comment:
this is soon to be fixed upstream - have tested with it symlinked
https://github.com/xtermjs/xterm.js/pull/3637/files#diff-2d8314c25a625261e947dd08a0c145425e19e8e82b5be8684a6b3fdefc9a5654R1295